src/content_scripts/applyColorSchemeCss.js

```diff
@@ -170,5 +170,5 @@ async function applyWantedStyle() { // eslint-disable-line no-unused-vars
     });
 }
 
-// apply style when DOM content is loaded
-document.addEventListener("DOMContentLoaded", applyWantedStyle);
+// apply style when content is fully loaded
+window.addEventListener('load', applyWantedStyle);
```

Comment:
please see eslint/code styles:

```suggestion
window.addEventListener("load", applyWantedStyle);
```